src/Data/MonadicStreamFunction/InternalCore.hs

```diff
@@ -88,23 +87,23 @@ instance Monad m => Category (MSF m) where
 -- types, and doesn't influence control flow.
 -- Other handling functions like exception handling or 'ListT' broadcasting
 -- necessarily change control flow.
-morphGS :: Monad m2
-        => (forall c . (a1 -> m1 (b1, c)) -> (a2 -> m2 (b2, c)))
+morphGS :: Functor m2
+        => (InMorph m1 a1 b1 ~> InMorph m2 a2 b2)
```

Comment:
I personally find this type synonym no readability improvement.

Reply:
I meant both, although I personally quite like `~>`. (But it's @ivanperez-keera you need to convince ;) I'm just saying my personal opinion) If `InMorph` is used elsewhere, it would be good style to introduce this synonym, but I don't think it's needed much.